tests/behat/src/FeatureContext.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace SilverStripe\LinkField\Tests\Behat;
+
+use Facebook\WebDriver\WebDriverBy;
+use PHPUnit\Framework\Assert;
+use SilverStripe\BehatExtension\Context\SilverStripeContext;
+
+class FeatureContext extends SilverStripeContext
+{
+    /**
+     * @Given /^I focus on the "([^"]+)" element$/
+     */
+    public function theHasALinkInLinkField(string $selector)
+    {
+        $page = $this->getSession()->getPage();
+        $element = $page->find('css', $selector);
+        Assert::assertNotNull($element, sprintf('Element %s not found', $selector));
+        $element->focus();
+    }
+
+    /**
+     * @Given /^I type "([^"]+)" in the field$/
+     */
+    public function iTypeInField(string $data)
```

Comment:
```suggestion
    public function iTypeInTheField(string $data)
```
Method name should match Given regex

This method should be put into behat-extension BasicContext

Reply:
Updated. Moved in `behat-extension`. [See](https://github.com/silverstripe/silverstripe-behat-extension/pull/263/files)